app/models/ability.rb

```diff
@@ -0,0 +1,35 @@
+class Ability
```

Comment:
- [Optional] What did you think of adding a line of code for the guest user?

```.ruby
   def initialize(user)

        user ||= User.new
      # Define abilities for the passed in user here. For example:
        if user.is? :admin
        can :manage, :all
        ..
        ..
 ```

Reply:
Thank you, I'll do that